tests/glsl/buffer_handle_1.spv.yaml

```diff
@@ -320,398 +320,3 @@ all_type_descriptions:
     members:
       - *td13
       - *td16
-  - &td18
```

Comment:
Looked, these were duplicates (these tests do recursion looping) so this is actually good it was removed